consensus/babe/types.go

```diff
@@ -42,3 +42,12 @@ type AuthorityData struct {
 	id     *sr25519.PublicKey
 	weight uint64
 }
+
+// BabeHeader as defined in Polkadot RE Spec, definition 4.10 in section 4.1.4
+type BabeHeader struct {
+	Slot          uint64
+	VRFOutput     [32]byte
+	VRFProof      [32]byte
+	BlockProducer *sr25519.PublicKey
+	Signature     []byte
```

Comment:
I don't think there is a signature field?

Reply:
nvm, I see it now

Reply:
Isn't the symbol Sb for a signature of Block Header?

Reply:
yes, but what I'm seeing in the spec is that the signature isn't part of the BABE header, it's part of the block digest, are you on the most recent version of the spec? it's definition 5.10 for me

Reply:
Oh my, no I was looking at the wrong spec, sorry everyone!  Browser bookmarked to github.com/w3f/polkadot-re-spec which now I see is deprecated.  I'll look at the current on at github.com/w3f/polkadot-spec.

Reply:
no worries :)